xcode/Subconscious/Shared/AppTheme.swift

```diff
@@ -66,10 +66,9 @@ extension Color {
     static let placeholderText = SwiftUI.Color("PlaceholderTextColor")
     static let textPressed = text.opacity(0.5)
     static let textDisabled = placeholderText
-    static let secondaryText = SwiftUI.Color("SecondaryTextColor")
     static let tertiaryText = placeholderText
     static let icon = SwiftUI.Color.accentColor
-    static let secondaryIcon = SwiftUI.Color.secondaryText
+    static let secondaryIcon = SwiftUI.Color.secondary
```

Comment:
SwiftUI now exposes secondary color via SwiftUI.Color. We should use that instead.